src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php

```diff
@@ -266,6 +266,20 @@ protected function render(string $view, array $parameters = [], Response $respon
         return $response;
     }
 
+    /**
+     * Renders a view for a form.
+     */
+    protected function renderForm(string $view, FormInterface $form, array $parameters = [], Response $response = null): Response
+    {
+        $response = $this->render($view, $parameters, $response);
```

Comment:
`['form' => $form->createView()] + $parameters`

Reply:
Replied there: https://github.com/symfony/symfony/pull/41178#issuecomment-839497167

```
[$formVaribaleName => $form->createView()] + $parameters
```

seems better, isn't?

Reply:
What about `renderForm(string $view, array $parameters = [], $formIndex = 'form', Response $response = null)`,
and we throw if $parameters[$formIndex] is not a FormInterface?

Reply:
`renderForm(string $view, FormInterface $form, array $parameters = [], string $formIndex = 'form', Response $response = null)` works for me also :)

Reply:
I took the second option :)